geode-experimental-driver/src/main/java/org/apache/geode/experimental/driver/ProtobufDriver.java

```diff
@@ -44,10 +47,7 @@
    */
   private final Set<InetSocketAddress> locators;
```

Comment:
I don't think is needed as a class member any more since it's only passed into the channel.